resim/auth/device_code_client.cc

```diff
@@ -46,6 +46,7 @@ HttpResponse json_query(
       cpr::Header{{"Content-Type", "application/json"}},
       cpr::Body{request_json});
 
+  response->Clear();
```

Comment:
Associated with newer versions of protobuf. This was causing an infinite loop because parsing no longer overwrote the "error" field in the message.